ci-operator/step-registry/mtr/retrieve-cluster-url/mtr-retrieve-cluster-url-commands.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Get the apps URL and pass it to env.sh for the mtr-runner container to use
+URL=$(oc get route -n openshift-console console -o jsonpath='{.spec.host}')
```

Comment:
What us the main reason that you decided to break this out as its own ref? Rather then just doing this as part of the execute-interop-ui-tests ref?

I'm asking because I'm doing this same thing in ACM's [clc tests commands.sh](https://github.com/openshift/release/pull/36326/files#diff-847aa9ab8a0dc92f788d7384f0f9139d5e053ec4d9c116e1c3b4b344fae5aa84R16) along with some other dynamic vars that are needed.

Splitting this out would give us the ability to just use it infront of another test that may need the url that might not be the interop test. In my model, if another test ref needs the URL I'll need to copy and paste that code into this new test ref.

Reply:
@ascerra I am conflicted on this one.  My original reason was really because I wasn't sure that this one thing was enough to add the oc client to the execution container. But then again, I'm not sure it's enough for it's own ref. If we will need this in another scenario, I feel better about adding it to our own toolset for reuse.

I'm okay with either

Reply:
Resolved in commit [4602739](https://github.com/openshift/release/pull/36181/commits/4602739b55370b75da05c3fc8352787545510f44)

Ended up being able to build this value using existing environment variables.